config/locales/en.yml

```diff
@@ -353,39 +353,28 @@ en:
   # homepage translations
   home:
     index:
-      title: "Welcome to the UK Parliament's beta website"
-      new_website: "We're building a %{link}. Pages with a beta label are being tested and improved. They may not work for everyone."
-      new_website_link: 'new website for Parliament'
-      old_website: 'Beta pages are separate from the current %{link} website.'
-      old_website_link: 'parliament.uk'
-      feedback:
-        title: 'Give feedback'
-        provide: 'We want your feedback to %{link} further.'
-        provide_link: 'help us improve these pages'
+      title: 'UK Parliament beta website'
+      new_website: "We're building a new website for Parliament. Pages with a beta label are being tested and improved. They may not work for everyone."
```

Comment:
Do you want to change this to single quotes and escape the apostrophe, so it's inline with the others?

Reply:
Unfortunately there is the 'We're' problem which would escape the string hence double quotes.